streams-activitypub-servlets/src/main/scala/org/apache/streams/activitypub/servlets/WebfingerServlet.scala

```diff
@@ -42,6 +30,11 @@ import scala.util.Try
  * as the default when no Accept is supplied, so that casual use through a
  */
 
+object WebfingerServlet {
+  given webfinger : WebfingerApi = WebfingerGraphImpl.DEFAULT
+
```

Comment:
WebfingerGraphImpl.DEFAULT will be used by the servlet to fulfill 
`WebfingerApi.webfingerQuery(request: WebfingerQueryRequest): WebfingerQueryResponse`
unless another implementation is 'given' in a narrower scope than this, or is passed as an implicit ('using' in scala 3) to the servlets doWebfingerQuery method